scripts/globals/spells/bluemagic/disseverment.lua

```diff
@@ -1,4 +1,4 @@
------------------------------------
+-----------------------------------------
```

Comment:
you should avoid any extra changes like this for the time being. just adds to an already large diff to read.